daras_ai_v2/azure_doc_extract.py

```diff
@@ -54,7 +56,17 @@ def azure_form_recognizer_models() -> dict[str, str]:
     return {value["modelId"]: value["description"] for value in r.json()["value"]}
 
 
+def azure_should_retry(e: Exception) -> bool:
+    import requests
+
+    return isinstance(e, requests.exceptions.RequestException) and (
+        (hasattr(e, "response"))
+        or e.response.status_code in [408, 429]  # Retry for rate limits or timeouts
+    )
```

Comment:
<img width="915" alt="image" src="https://github.com/user-attachments/assets/8ebdb90e-3e9c-47e4-88d7-fc936f924ee8">
??? not sure what you are trying to do here

Reply:
Thanks for pointing this out! I accidentally pushed the wrong version of the code and will correct it. I should have been more cautious.